Return type annotation for `get()` is too restrictive

Attempting to call `cache.get()` on anything that does not return a string results in a type error when running mypy:

```py
class CustomClass:
    pass

def get_cached_class() -> Optional[CustomClass]:
    return cache.get("key")

$ mypy test.py
> error: Incompatible return value type (got "Optional[str]", expected "Optional[CustomClass]")  [return-value]
```

If we take a look at the type in cachelib (to which this method simply proxies), it actually returns `typing.Any` (which makes sense, because the implementation handles serialization/deserialization using pickle):

https://github.com/pallets-eco/cachelib/blob/c00d97f757364ad99e1fd31b14f5f8f5aacd714c/src/cachelib/base.py#L21

A simple fix would be to just swap out the return type for `Any`.

Environment:

- Python version: 3.8
- Flask-Caching version: 2.0.2

--- a/src/flask_caching/__init__.py
+++ b/src/flask_caching/__init__.py
@@ -15,18 +15,13 @@ import logging
 import uuid
 import warnings
 from collections import OrderedDict
-from typing import Any
-from typing import Callable
-from typing import Optional
-from typing import Tuple
-from typing import Union
+from typing import Any, Dict, List, Callable, Optional, Tuple, Union
 
 from flask import current_app
 from flask import Flask
 from flask import request
 from flask import Response
 from flask import url_for
-from markupsafe import Markup
 from werkzeug.utils import import_string
 
 from flask_caching.backends.base import BaseCache
@@ -185,7 +180,7 @@ class Cache:
         app = current_app or self.app
         return app.extensions["cache"][self]
 
-    def get(self, *args, **kwargs) -> Optional[Union[str, Markup]]:
+    def get(self, *args, **kwargs) -> Any:
         """Proxy function for internal cache object."""
         return self.cache.get(*args, **kwargs)
 
@@ -205,7 +200,7 @@ class Cache:
         """Proxy function for internal cache object."""
         return self.cache.delete(*args, **kwargs)
 
-    def delete_many(self, *args, **kwargs) -> bool:
+    def delete_many(self, *args, **kwargs) -> List[str]:
         """Proxy function for internal cache object."""
         return self.cache.delete_many(*args, **kwargs)
 
@@ -217,15 +212,15 @@ class Cache:
         """Proxy function for internal cache object."""
         return self.cache.get_many(*args, **kwargs)
 
-    def set_many(self, *args, **kwargs):
+    def set_many(self, *args, **kwargs) -> List[Any]:
         """Proxy function for internal cache object."""
         return self.cache.set_many(*args, **kwargs)
 
-    def get_dict(self, *args, **kwargs):
+    def get_dict(self, *args, **kwargs) -> Dict[str, Any]:
         """Proxy function for internal cache object."""
         return self.cache.get_dict(*args, **kwargs)
 
-    def unlink(self, *args, **kwargs) -> bool:
+    def unlink(self, *args, **kwargs) -> List[str]:
         """Proxy function for internal cache object
         only support Redis
         """
@@ -474,7 +469,7 @@ class Cache:
 
                 return cache_key
 
-            def _make_cache_key(args, kwargs, use_request):
+            def _make_cache_key(args, kwargs, use_request) -> str:
                 if query_string:
                     return _make_cache_key_query_string()
                 else:

